.env.template

```diff
@@ -1,3 +1,6 @@
 # Azure Form Recognizer
 FORMRECOGNIZER__APIKEY=00000000000000000000000000000000
 FORMRECOGNIZER__ENDPOINT=https://westus2.api.cognitive.microsoft.com/
```

Comment:
We should probably change this Endpoint url to make it clear that this is not the default and must be specified